README.md

```diff
@@ -1,6 +1,7 @@
 # Zenko Backbeat
 
 ![backbeat logo](res/backbeat-logo.png)
+[![Build Status][badgepriv]][badgepub]
```

Comment:
these badge are not correct, the URLs (at the end of this file) should be updated